### DISCUSSION TITLE
타운홀 가는 문 바로 앞에서 poke로 사람을 불러서 텔레포트 하면 어디로 갈까요?

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#22] [제보하기]
1. 아래 그림과 같은 위치에서
2. 다른 사용자를 poke로 호출
3. 다른 사용자가 텔레포트로 이동하면 어디로 이동될까요?

=> 현재는 타운홀로 이동합니다. 현재 스페이스로 불렀는데, 타운홀로 이동되는 케이슨데, 결함으로 보는게 맞을까요?

<img width="320" alt="image" src="https://github.com/metamong-friends/metamong-friends.github.io/assets/24870098/c87b1d55-044f-4d9d-8fe6-4ddb061e8273">

**Comment by ACT-TJ** [comment]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 고칠까요?? 즐거워보이셨는데.

**Reply by shlee303** [reply]
와악! 재밌네요 ㅋㅋㅋㅋㅋ
